src/main/java/org/cryptomator/frontend/fuse/FuseNioAdapter.java

```diff
@@ -6,6 +6,19 @@
 
 	boolean isMounted();
 
+	/**
+	 * Checks if it is not safe to unmount this filesystemin.
+	 *
+	 * Important: This function only checks if it is _not_ safe and a return value of {@code false} should not be considered as an unmount can be safely executed.
+	 * The file system may be in state where it cannot be determined, hence this method returns false.
+	 *
+	 * API note: It is the task of the fs developer to decide what "not safe" means. For example, "not safe" can mean there are open resources or pending operations.
+	 * Additionally no guarantees about the validity of the result after the call is made, i.e. the result may be immediately outdated.
+	 *
+	 * @return true if the filesystem is in a state, where an unmount should not be performed.
+	 */
+	boolean isNotSafeToUnmount();
```

Comment:
This is a highly specific method name, suggesting it is only useful for one purpose (i.e. in preparation of unmount). I'd prefer a more generic name, if this is exposed via an public API rather than an unmount-specific internal behaviour.

I'd suggest `isInUse` as it is easy to understand that returning "false" does not make any promises that it is _not_ in use.

Reply:
Done with c4e741f832c42f6ffa99e8af5ab11eb4b9b5c65b